name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/YahooFinanceQuoteFeed.java

```diff
@@ -206,8 +206,8 @@ else if (days < 600)
         else if (days < 1500)
             range = "5y"; //$NON-NLS-1$
 
-        return new WebAccess("query1.finance.yahoo.com", "/v7/finance/spark") //
-                        .addParameter("symbols", security.getTickerSymbol()).addParameter("range", range)
+        return new WebAccess("query1.finance.yahoo.com", "/v7/finance/chart" + security.getTickerSymbol()) //
```

Comment:
Thanks. It works for me if I add a dash "/" after the chart path element

Reply:
And then it looks to me that the API is now using "v8" as version identifier. So I bumped that up too.